demoscene/utils/groklinks.py

```diff
@@ -1531,14 +1531,26 @@ def __unicode__(self):
         (domain, name) = self.param.split('/')
         return u"https://%s.bandcamp.com/track/%s" % (domain, name)
 
+def TwitchChannel(BaseUrl):
```

Comment:
Should be `class`, not `def`. If you can, please set up a development environment so that you can test these things before opening a PR!